models/experimental/functional_stable_diffusion/tests/test_perf_stable_diffusion.py

```diff
@@ -77,6 +77,7 @@ def lms_derivative(tau):
 
 @skip_for_grayskull()
 @pytest.mark.models_performance_bare_metal
+@pytest.mark.parametrize("device_l1_small_size", [24576], indirect=True)
```

Comment:
main sd test `tests/ttnn/integration_tests/stable_diffusion/test_unet_2d_condition_model.py ` uses 32k, we should keep it consistant.

Reply:
unless you're changing default to 24k